x-pack/legacy/plugins/siem/public/containers/matrix_histogram/utils.ts

```diff
@@ -71,7 +71,7 @@ export const useQuery = <Hit, Aggs, TCache = object>({
       return apolloClient
         .query<GetMatrixHistogramQuery.Query, GetMatrixHistogramQuery.Variables>({
           query,
-          fetchPolicy: 'cache-first',
+          fetchPolicy: 'network-only',
```

Comment:
another great catch, as this would prevent the `Update` button from refreshing the content, depending on how things were wired up "above" the component